public/js/script.js

```diff
@@ -236,7 +444,7 @@ $('textarea').keyup(function(e) {
     }
 
     else{
-        $('.terminal-output').append('<div class="command" role="presentation" aria-hidden="true"><div style="width: 100%;"><span class="user">root@' + username + ': ~$ </span><span>' + command + '</span></div></div>');
+        $('.terminal-output').append('<div class="command" role="presentation" aria-hidden="true"><div style="width: 100%;"><span class="user">root@' + username + ': ~/' + directory + '$ </span><span>' + command + '</span></div></div>');
```

Comment:
Here also I think we need the if(directory=="") part.

Reply:
I made certain changes that u suggested me !!